samples/dashboard/demos/climate/demo.html

```diff
@@ -1,10 +1,10 @@
+<link rel="stylesheet" type="text/css" href="https://code.highcharts.com/css/dashboard/gui.css">
+
 <script src="https://code.highcharts.com/dashboard.js"></script>
 <script src="https://code.highcharts.com/datagrid.js"></script>
 <script src="https://code.highcharts.com/highcharts.js"></script>
 <script src="https://code.highcharts.com/modules/dashboard-plugin.js"></script>
-<script src="https://code.highcharts.com/modules/data.js"></script>
 <script src="https://code.highcharts.com/modules/map.js"></script>
 <script src="https://code.highcharts.com/modules/timeline.js"></script>
-<script src="https://code.highcharts.com/mapdata/custom/world.js"></script>
 
-<div id="container"></div>
+<div id="container">Loading...</div>
```

Comment:
This should be hidden when demo is ready.

Reply:
Yes, the Dashboard should reset the container like Highcharts does.

Reply:
Moved to Asana